athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java

```diff
@@ -167,7 +167,7 @@ private void buildGraphTraversal() {
                 Vertex vertex1 = tinkerGraph.addVertex(T.label, "default");
                 vertex1.property("property1", 10);
                 vertex1.property("property2", "string1");
-                vertex1.property("property3", 12.4);
+                vertex1.property("property3", 12);
```

Comment:
why this change? can you add another property instead of changing existing?

Reply:
Also, the bugs that we found are:

1. Failure to detect null values returned from the data store and handling those gracefully before returning the result set to Athena.
2. Type casting issues for certain data types such as Double.

Reply:
i'm not sure this answers my question of why you removed the 12.4 test case as opposed to adding a new one.

Reply:
added new test cases